blocks/core/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -1,38 +0,0 @@
-//header
```

Comment:
@roobottom Fair enough! Yeah looking more closely I do see that it's all pretty much theme specific stuff so that makes sense. In which case, yeah file should probably go.